MediaBrowser.Api/Playback/MediaInfoService.cs

```diff
@@ -613,18 +595,16 @@ private void SetDeviceSpecificSubtitleInfo(StreamInfo info, MediaSourceInfo medi
 
             foreach (var profile in profiles)
             {
-                foreach (var stream in mediaSource.MediaStreams)
+                foreach (var stream in mediaSource.MediaStreams
```

Comment:
Again, previous one was cleaner imo

Reply:
Agreed. Alternatively, I would be okay defining a new variable above to take this out of the `for` expression. Additionally the `&&` should only be indented once.